src/pools/pool.service.ts

```diff
@@ -136,6 +137,7 @@ export class PoolService {
       this.pool.lastUpdate = Date.now();
     }
 
+    this.pool.maxApr = Number(this.absMaxApr(poolApr, this.pool.boost));
```

Comment:
check this please @garethfuller @timjrobinson if I am right here

Reply:
What is the boost field on a pool? We can't calculate the APR on a per user basis in the API. Are there some pools  that can be boosted and others that can't and if so we can do that, but I thought that would be applied to poolApr.max already.

Reply:
@garethfuller can you help here please. I've taken it from current sorting logic: https://github.com/balancer-labs/frontend-v2/blob/6c95afd0f701c283ccb2461cfc2de5b700cf7e65/src/composables/usePool.ts#L228